docs/index.md

```diff
@@ -0,0 +1,159 @@
+# OpenFF Units
+
+Units of measure for biomolecular software.
+
+OpenFF Units is based on [Pint]. It's [`Quantity`], [`Unit`], and [`Measurement`] types inherit from Pint's, and add improved support for serialization and deserialization. OpenFF Units improves support for biomolecular software by providing a [system of units] that are compatible with [OpenMM] and [providing functions] to convert to OpenMM units and back. It also provides [atomic masses] with units, as well as some [other useful maps].
+
+[system of units]: openff.units.unit
+[OpenMM]: openmm:index
+[Pint]: pint:index
+[`Quantity`]: openff.units.Quantity
+[`Unit`]: openff.units.Unit
+[`Measurement`]: openff.units.Measurement 
+[providing functions]: openff.units.openmm
+[atomic masses]: openff.units.elements.MASSES
+[other useful maps]: openff.units.elements
+
+## Installation
+
+We recommend installing OpenFF Units with the [Conda] package manager. If you don't yet have a Conda distribution installed, we recommend [MambaForge] for most users. The `openff-units` package can be installed from Conda Forge:
+
+```shell
+conda install -c conda-forge openff-units
+```
+
+[Conda]: https://conda.io
+[MambaForge]: https://github.com/conda-forge/miniforge#mambaforge
+
+## Using OpenFF Units
+
+OpenFF Units provides the [`Quantity`] class, which represents a numerical value with units. To create a `Quantity`, multiply a value by the appropriate unit:
+
```

Comment:
I find myself using the explicit construction
```
mass_proton = unit.Quantity(1.007, unit.amu)
# or even more explicit, maybe TOO explicit
mass_proton = unit.Quantity(value=1.007, units=unit.amu)

```

from time to time because it avoids long expressions on one line, particularly when using arrays and long (i.e. kcal/mol) units.. Including this in the docs would make it a bit more explicit what object users are getting here, i.e. introducing it like this and then "but you can also just use a `*` operator ...." and stick with the shorthand.

If this is too much too fast for an introduction, I'd still like to see something like

```python
>>> # oh by the way, here's the type of the object we made...
>>> type(mass_proton)
openff.units.units.Quantity
```